src/index.ts

```diff
@@ -217,7 +227,7 @@ const notebooks: JupyterFrontEndPlugin<void> = {
  */
 const main: JupyterFrontEndPlugin<IDebugger> = {
   id: '@jupyterlab/debugger:main',
-  optional: [ILayoutRestorer, ICommandPalette],
+  optional: [ILayoutRestorer, ICommandPalette, ILabShell],
```

Comment:
Should we make `ILabShell` part of the `requires`? (similar to the other plugins)

Reply:
Good point, I resolved that.

Reply:
Thanks!